tests/test_load_metadata.c

```diff
@@ -151,6 +174,31 @@ static void test_cr_metadata_locate_and_load_xml_detailed(void)
 }
 
 
+#ifdef WITH_LIBMODULEMD
+static void test_cr_metadata_locate_and_load_modulemd(void)
+{
+    int ret;
+    guint size;
+    cr_Package *pkg;
+    cr_Metadata *metadata;
+
+    metadata = cr_metadata_new(CR_HT_KEY_NAME, 0, NULL);
+    g_assert(metadata);
+    ret = cr_metadata_locate_and_load_xml(metadata, TEST_REPO_03, NULL);
+    g_assert_cmpint(ret, ==, CRE_OK);
+    size = g_hash_table_size(cr_metadata_hashtable(metadata));
+    g_assert_cmpuint(size, ==, REPO_SIZE_00);
```

Comment:
Should be REPO_SIZE_03, but other than this small detail it all looks good to me.